src/platform/site-wide/announcements/components/AllClaimsBetaBanner.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import environment from '../../../utilities/environment';
+
+export default function AllClaimsBetaBanner({ dismiss }) {
+  // staging only. Remove before launch
+  if (environment.isProduction()) return null;
```

Comment:
Is the idea here to merge this PR in as-is and then remove this this in a separate PR when we're ready to launch beta?

Reply:
Yes—the PR as it stands will always show the banner so we can see what it looks like in staging

Reply:
We will also need to change the URL before launch if it changes